core/server/common/src/main/java/alluxio/Registry.java

```diff
@@ -149,6 +149,16 @@ public void stop() throws IOException {
     }
   }
 
+  /**
+   * Closes all {@link Server}s in reverse dependency order. If A depends on B, B is closed
+   * before A.
```

Comment:
Is this accurate? Shouldn't we close A before B?

Reply:
Code does the right thing, but the doc was wrong.